docs/DeveloperGuide.md

```diff
@@ -126,7 +126,7 @@ The `Model` component,
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
+<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Interest` list in the `AddressBook`, which `InterestsList` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `InterestsList` needing their own `Interest` objects.<br>
```

Comment:
For the line **This allows `AddressBook` to only require one `Tag` object per unique tag**, shouldnt it be one `Interest` object per unique Interest?